common/djangoapps/student/admin.py

```diff
@@ -1,11 +1,31 @@
 '''
 django admin pages for courseware model
 '''
+from django import forms
 from config_models.admin import ConfigurationModelAdmin
 
 from student.models import UserProfile, UserTestGroup, CourseEnrollmentAllowed, DashboardConfiguration
-from student.models import CourseEnrollment, Registration, PendingNameChange, CourseAccessRole, CourseAccessRoleAdmin
+from student.models import CourseEnrollment, Registration, PendingNameChange, CourseAccessRole
 from ratelimitbackend import admin
+from student.roles import REGISTERED_ACCESS_ROLES
+
+
+class CourseAccessRoleForm(forms.ModelForm):
+    """Form for adding new Course Access Roles view the Django Admin Panel."""
+    class Meta:
+        model = CourseAccessRole
+
+    COURSE_ACCESS_ROLES = [(role_name, role_name) for role_name in REGISTERED_ACCESS_ROLES.keys()]
+    role = forms.ChoiceField(choices=COURSE_ACCESS_ROLES)
```

Comment:
The way the roles are persisted is one row per role / user / course pairing. If you are staff, sales_admin, and finance_admin for a single course, you would have three rows.